README.md

```diff
@@ -96,7 +96,7 @@ It includes advanced usage like how to run an additional server to support tests
 
 Cross-browser Testing Platform and Open Source ♥ Provided by [Sauce Labs](https://saucelabs.com).
 
-[![Sauce Labs logo](./doc/images/sauce-labs-logo.svg)](https://saucelabs.com)
+[![Sauce Labs logo](./doc/images/sauce-labs-logo.svg)](https://saucelabs.com) [![Greenkeeper badge](https://badges.greenkeeper.io/airtap/airtap.svg)](https://greenkeeper.io/)
 
```

Comment:
And we don't want this badge either. Kind of annoying that they PR that actually.

Reply:
Yeah I always tweak the Greenkeeper PR to remove the badge